pandas/core/ops/docstrings.py

```diff
@@ -374,10 +384,12 @@ def _make_flex_doc(op_name, typ):
 Returns
 -------
 {series_returns}
+"""
 
+_see_also_reverse_SERIES = """
 See Also
 --------
-Series.{reverse}
+Series.{reverse} : {see_also_desc}.
```

Comment:
For now, can you undo the rest of the changes, and just leave something generic like this?

```suggestion
Series.{reverse} : Reverse of the operator, see `Python documentation <https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types>__` for more details.
```

This fixes the docstring validation problem, adds useful information to the user, and keeps things simple.

Reply:
There will still be a problem with comparision operators rendering `Series.None`. I really can't see a way to handle both algebraic and comparision operators with one template.